app/src/pages/inside/testCaseLibraryPage/testCaseDetailsPage/editTestCaseModal/useUpdateTestCase.ts

```diff
@@ -84,10 +86,27 @@ export const useUpdateTestCase = () => {
       });
 
       onSuccess();
+
+      const offset = testCasesPageData
+        ? (testCasesPageData?.number - 1) * testCasesPageData?.size
+        : 0;
+      const limit = testCasesPageData?.size;
+
       if (payload.folderId) {
-        dispatch(getTestCaseByFolderIdAction({ folderId: Number(payload.folderId) }));
+        dispatch(
+          getTestCaseByFolderIdAction({
+            folderId: Number(payload.folderId),
+            offset,
+            limit,
```

Comment:
offset and limit can be stored in one object and used here and in `getAllTestCasesAction`